Resources/doc/installation.rst

```diff
@@ -78,6 +78,19 @@ the MongoDB ODM across your application:
     Of course, you'll also need to make sure that the MongoDB server is running
     in the background. For more details, see the MongoDB `Quick Start`_ guide.
 
+
+###Authorization
+
+If you use authorization on your mongo database you can provide username, password and authentication database in the following way:
```

Comment:
Please change all occurrences of "mongo" to "MongoDB". Also there should be a comma after "password"